compiler/rustc_codegen_llvm/src/allocator.rs

```diff
@@ -77,7 +77,7 @@ pub(crate) unsafe fn codegen(
                 llvm::LLVMRustGetOrInsertFunction(llmod, callee.as_ptr().cast(), callee.len(), ty);
             llvm::LLVMRustSetVisibility(callee, llvm::Visibility::Hidden);
 
-            let llbb = llvm::LLVMAppendBasicBlockInContext(llcx, llfn, "entry\0".as_ptr().cast());
+            let llbb = llvm::LLVMAppendBasicBlockInContext(llcx, llfn, c"entry".as_ptr().cast());
```

Comment:
can we avoid changing the compiler here already for some more time? just in case there may be further regressions or problems
I think the new beta bootstrap compiler will also not have support for it.

Reply:
yeah ive undone that